src/js/components/commits/list.tsx

```diff
@@ -97,13 +99,27 @@ TimestampTableCell.displayName = DataTableCell.displayName;
 const CommitList = ({ commits }: { commits: Commit[] }) =>
   commits.length ? (
     <>
-      <h2
-        className="slds-text-heading_medium
-          slds-m-top_large
-          slds-m-bottom_x-small"
-      >
-        Commit History
-      </h2>
+      <div className="slds-is-relative heading">
```

Comment:
@dvdherron Maybe this is over-engineering, but I wonder if it would be cleaner for the `<TourPopover>` component itself to render its own `<div className="slds-is-relative">` container, and to also accept an optional `className` prop to add e.g. `className="heading"` or whatever. It would sure make the diff simpler from the places where we're adding in popovers... but I don't know if a single wrapping `div` would work in every case?

Reply:
Oh, I guess that wouldn't really work because the popover _and_ some arbitrary contents need to be wrapped together... Probably simpler to just leave it as-is then 👍

Reply:
yeah, inserting it to each respective reference element probably has to be done manually with this type of popover . . .